app/backend/wells/migrations/0067_geom_updates.py

```diff
@@ -9,16 +9,18 @@
 def update_fields(apps, schema_editor):
     activitySubmission = apps.get_model('wells', 'activitysubmission')
     try:
+        activitySubmission._meta.get_field('longitude')
         activitySubmission._meta.get_field('latitude')
         schema_editor.execute("update activity_submission set geom = ST_SetSrid(ST_MakePoint(longitude, latitude), 4326);")
-    except FieldDoesNotExist:
+    except Exception:
```

Comment:
If you don't specify that you're catching an Exception, you can catch an even broader number of issues.
```suggestion
    except:
```